main.go

```diff
@@ -59,10 +62,24 @@ var (
 	version string
 	date    string
 	c       *config
+	client  *http.Client
 )
 
+func ConfigureClient() {
```

Comment:
__[Golint]__ exported function ConfigureClient should have comment or be unexported ([link](https://sider.review/gh/repos/51703546/pulls/29#issue-4599797))

<sub>posted by Sider</sub>